client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js

```diff
@@ -461,6 +461,125 @@ export default class {
         ));
   }
 
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:tucVoipServiceAlias#fetchContactCenterSolutionNumberScreenListsStatus
+   *  @methodOf managerApp.service:tucVoipServiceAlias
+   *
+   *  @description
+   *  <p>Fetch screen lists current status for a contact center solution number.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   */
+  fetchContactCenterSolutionNumberScreenListsStatus({ billingAccount, serviceName }) {
+    return this.OvhApiTelephony.EasyHunting().ScreenListConditions().v6().get({
+      billingAccount,
+      serviceName,
+    }).$promise;
+  }
+
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:tucVoipServiceAlias#
+   *  fetchContactCenterSolutionNumberScreenListsConditions
+   *  @methodOf managerApp.service:tucVoipServiceAlias
+   *
+   *  @description
+   *  <p>Fetch screen lists for a contact center solution number.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   *
+   *  @return {Array[Object]}                     Screen lists
+   */
+  fetchContactCenterSolutionNumberScreenListsConditions({ billingAccount, serviceName }) {
+    return this.OvhApiTelephony.EasyHunting().ScreenListConditions().Conditions().v6()
+      .query({
+        billingAccount,
+        serviceName,
+      }).$promise
+      .then(ids => this.$q
+        .all(
+          _.chunk(ids, 50).map(chunkIds => this.OvhApiTelephony.EasyHunting()
+            .ScreenListConditions().Conditions().v6()
+            .getBatch({
+              billingAccount,
+              serviceName,
+              conditionId: chunkIds,
+            }).$promise),
+        )
+        .then(chunkResult => _(chunkResult).flatten().pluck('value').value()));
```

Comment:
```suggestion
        .then(chunkResult => _(chunkResult).flatten().map('value').value()));
```
instead ? (`pluck` no longer exists in lodash4)